packages/node_modules/@webex/webex-core/src/interceptors/auth.js

```diff
@@ -50,64 +50,58 @@ export default class AuthInterceptor extends Interceptor {
   }
 
   /**
-   * Determines if the provided options object needs an auth header
+   * Determines if the provided options object needs an authorization header.
+   *
    * @param {Object} options
    * @returns {Promise<boolean>}
    */
   requiresCredentials(options) {
-    if (options.service === 'u2c') {
-      if (options.resource.includes('limited')) {
+    // Validate that authorization is necessary.
+    if (options.addAuthHeader === false) {
```

Comment:
Could we do the opposite here as well? If addAuthHeader is true, resolve true?

Reply:
If this interceptor is to match the same output when intercepting, we can not. If the `addAuthHeader` option is anything but false, it will check to confirm it's a service and then check to confirm if it's in the whitelist before adding the auth header.